pinax/api/jsonapi.py

```diff
@@ -43,7 +46,7 @@ def from_validation_error(cls, exc, resource_class):
                 errs.append(err)
         return cls(errors=errs)
 
-    def __init__(self, data=None, errors=None, links=False, included=None, meta=None, linkage=False):
+    def __init__(self, data=None, errors=None, links=False, included=None, meta={}, linkage=False):
```

Comment:
Another fix is to leave it `None` and revert the behavior in `serializable`. Is there another reason I am not seeing for this change?